<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerModelMirrorPlusModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerModelMirrorPlusModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QRadioButton" name="CustomPlaneButton">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Flipping Plane:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="XMinButton">
          <property name="text">
           <string>X Min</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>InputModel:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Output Model:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="YMinButton">
          <property name="text">
           <string>Y Min</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QRadioButton" name="ZMinButton">
          <property name="text">
           <string>Z Min</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="XMaxButton">
          <property name="text">
           <string>X Max</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QRadioButton" name="YMaxButton">
          <property name="text">
           <string>Y Max</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QRadioButton" name="ZMaxButton">
          <property name="text">
           <string>Z Max</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1" colspan="4">
         <widget class="qMRMLNodeComboBox" name="OutputModelSelector">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="qMRMLNodeComboBox" name="InputModelSelector">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="renameEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="4">
         <widget class="QCheckBox" name="MergeMirrorInputCheckbox">
          <property name="text">
           <string>Merge mirror model with input model</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ApplyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerModelMirrorPlusModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputModelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerModelMirrorPlusModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>OutputModelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
